pVAC-Seq.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+from subprocess import call
+import os
+
+def pvac_path_for_script(script):
+    pVac_directory = os.path.abspath(os.path.dirname(__file__))
+    return os.path.join(pVac_directory, "pvac_seq", script)
+
+def run_subcommand(script, args):
+    script_cmd = "%s %s %s" % (
+        sys.executable,
+        pvac_path_for_script(script),
+        " ".join(args)
+    )
+    call([script_cmd], shell=True)
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    subparsers = parser.add_subparsers()
+
+    #add subcommands
+    variant_sequences_parser = subparsers.add_parser("generateVariants",
+                                                     help="Generates a variant peptide FASTA file from the TSV input file",
+                                                     add_help=False)
+    variant_sequences_parser.set_defaults(script="generate_variant_sequences.py")
+
+    binding_filter_parser = subparsers.add_parser("bindingFilter",
+                                                  help="Filters variants processed by NetMHC",
+                                                  add_help=False)
+    binding_filter_parser.set_defaults(script="binding_filter.py")
+
+    fasta_key_parser = subparsers.add_parser("generateKey",
+                                             help="Generates a FASTA key file",
+                                             add_help=False)
+    fasta_key_parser.set_defaults(script="generate_fata_key.py")
+
+    parse_netmhc_parser = subparsers.add_parser("parseNetMHC",
+                                                help="Parses output from NetMHC",
+                                                add_help=False)
+    parse_netmhc_parser.set_defaults(script="parse_output_netmhc.py")
+
+    coverage_filter_parser = subparsers.add_parser("coverageFilter",
```

Comment:
I think we should just leave this subcommand out for now until it has been converted for python unless there is a way to actually call the perl command here. @jasonwalker80 do you agree that this is how we should handle to two scripts that are still in perl?